tally_ho/apps/tally/forms/create_center_form.py

```diff
@@ -0,0 +1,42 @@
+from django import forms
+from django.forms import ModelForm, Textarea
+from tally_ho.apps.tally.models.center import Center
+
+disable_copy_input = {
+    'onCopy': 'return false;',
+    'onDrag': 'return false;',
+    'onDrop': 'return false;',
+    'onPaste': 'return false;',
+    'autocomplete': 'off',
+    'class': 'form-control'
+}
+
+
+class CreateCenterForm(ModelForm):
+    MANDATORY_FIELDS = ['name', 'office', 'code', 'tally']
```

Comment:
I actually think this is a left-over from a previous django version, seems to have no affect in QA

Reply:
To adjust this I've adjusted the `blank` setting in the model fields

Reply:
ahh, I see, OK looking through the existing code, this was done in a hacky way of overriding the `__init__` method and then looking at the `MANDATORY_FIELDS` value to adjust the required attribute of the internal fields. I think we'd be better off replacing this throughout with setting `blank` in the models